specs/DisableCrashReporting.md

```diff
@@ -0,0 +1,120 @@
+Custom Crash Reports
+===
+
+# Background
+Currently, when a WebView2 is created a WebView2 crashpad process will be created in addition to starting a WebView2 browser process. If any WebView2 process crashes, the crashpad process will create crash dumps and send them to Microsoft for diagnosis. This document covers new APIs to allow the end developer to customize crash reporting to help when running diagnostics and doing analysis. They can set the `CoreWebView2EnvironmentOptions.IsCustomCrashReportingEnabled` property to not have crash dumps sent to Microsoft and use the `CoreWebView2Environment.CrashDumpFolderPath` property to locate crash dumps and do customization with them instead.
```

Comment:
I think the passive voice of "to not have crash dumps sent to Microsoft" is confusing here.  Can we say "set this property to true to prevent crash dumps from being sent" so it's framed in terms of what change from default behavior the developer is requesting?

Do we expect this property to evolve for more "custom reporting" behaviors in the near future?  If not, why call is "custom" instead of naming the property "IsCrashDumpUploadBlocked"?

Reply:
Please update docs.

For naming see below.